help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,218 @@
+
+    * For instance, depending on the values you want to update, add the following mapping between your XDM profile schema and your [!DNL Dynamics 365] instance:
+        |XDM Profile Schema|[!DNL Dynamics 365] Instance|
+        |---|---|
+        |`person.name.firstName`|`FirstName`|
+        |`person.name.lastName`|`LastName`|
+        |`personalEmail.address`|`Email`|
+
+    * An example using these mappings is shown below:
+    ![Target mapping](../../assets/catalog/crm/microsoft-dynamics-365/mappings.png)
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.md) step you must manually map Platform segments to the custom field attribute in [!DNL Dynamics 365].
+
+To do this, select each segment, then enter the corresponding custom field attribute from [!DNL Dynamics 365] in the **[!UICONTROL Mapping ID]** field.
+
+> [!IMPORTANT]
+>
+> The value used for the Mapping ID should exactly match the name of the custom field attribute created within [!DNL Dynamics 365]. See [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) if you need guidance on finding your custom field attributes.
+
+An example is shown below:
+![Schedule segment export](../../assets/catalog/crm/microsoft-dynamics-365/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Browse Destinations](../../assets/catalog/crm/microsoft-dynamics-365/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Destinations Dataflow Run](../../assets/catalog/crm/microsoft-dynamics-365/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Destinations Activation Data](../../assets/catalog/crm/microsoft-dynamics-365/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Segment](../../assets/catalog/crm/microsoft-dynamics-365/segment.png)
+
+1. Log in to the Dynamics 365 website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You will also notice the segment status from Experience Platform has been updated against the corresponding custom field attribute that was provided in the **Mapping ID** field during the **[!UICONTROL Activate destination]** > **[!UICONTROL Schedule segment export]** step.
```

Comment:
Line 187: [...] during the [!UICONTROL Schedule segment export] step.

There is no  **[!UICONTROL Activate destination]** label anywhere in the UI.

Reply:
We are referring to the `https://github.com/AdobeDocs/experience-platform.en/blob/1b3618b96a31bbada9f74e9895b9d546e109c52a/help/destinations/assets/catalog/crm/microsoft-dynamics-365/schedule-segment-export.png` step. This corroboration between what was provided in the platform and what is visible in Dynamics 365 contacts page. 

Have prefixed the word Platform before **[!UICONTROL Activate destination]**.

Any suggestions on what rephrasing is needed ?

Reply:
We only use [!UICONTROL] for actual UI elements that are present in the Experience Platform UI (buttons, fields).
Please see the detailed information in our authoring guide on when to use these tags https://experienceleague.adobe.com/docs/authoring-guide-exl/using/authoring/localization/localize.html?lang=en.

In this section, there is no Activate destination button anywhere in the UI. The segment schedule step is already clearly documented, so you can link to that section.

As far as rephrasing suggestions go, please see my comment above. I have updated it with a clearer explanation.